xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/internal/html/DefaultHTMLCleaner.java

```diff
@@ -295,4 +308,19 @@ private CleanerTransformations getDefaultCleanerTransformations(HTMLCleanerConfi
 
         return defaultTransformations;
     }
+
+    /**
+     * @param configuration The configuration to parse.
+     * @return The HTML version specified in the configuration.
+     * @since 14.0RC1
+     */
+    private int getHTMLVersion(HTMLCleanerConfiguration configuration)
+    {
+        String param = configuration.getParameters().get(HTMLCleanerConfiguration.HTML_VERSION);
+        int htmlVersion = 4;
+        if (param != null) {
```

Comment:
would be better to call `!StringUtils.isBlank(htmlVersion)` no? Unless you're sure the string cannot be empty?

Reply:
Good point, though I guess we could alternatively also just do

```java
        if ("5".equals(param)) {
            htmlVersion = 5;
        }
```

as the only supported versions are 4 and 5. However, this ignores wrong values silently, no idea if this is okay or not.